parent/pom.xml

```diff
@@ -179,6 +179,12 @@
                 <version>${org.springframework.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
That was a fix for a dependency-version convergence that came up after adding spring as test-dep to the processor. Something like that. I'll check again.